### PR TITLE
Adjusts whitespace for a code block in AWS instructions.

### PR DESCRIPTION
**Pull request by arokem**
Mostly removes a lot of extraneous whitespace. The only "substantial" change is here: 

Screen Shot 2019-04-15 at 11 12 16 AM

### CODE REVIEW

File: doc/source/amazon/step-zero-aws.rst [hunk 1]
```diff
@@ -57,21 +57,25 @@ The Procedure
...
+
+    sudo apt-get update
+    sudo apt-get install awscli
 
 #. Set the availability zones for the nodes
 
    For this guide we will be allowing nodes to be deployed in all AZs::
 
        export ZONES=$(aws ec2 describe-availability-zones --region $REGION | grep ZoneName | awk '{print $2}' | tr -d '"')
-       export ZONES=$(echo $ZONES | tr -d " " | rev | cut -c 2- | rev)
```
Comment (betatim) [inline doc/source/amazon/step-zero-aws.rst]: Did you mean to delete this line?

Comment (betatim) [reply doc/source/amazon/step-zero-aws.rst]: I think yes from looking at the screenshot you posted.

**Comment by consideRatio** [comment]
LGTM thanks for your refinements @arokem!

For reference, do you understand what problems were caused by the additional processing of the ZONES bash variable, and why we one no longer need to do such processing?

File: doc/source/amazon/step-zero-aws.rst (hunk 1 above)
Comment (arokem) [reply doc/source/amazon/step-zero-aws.rst]: Yes. Before this line was removed: 

```
ubuntu@ip-172-31-94-142:~$ echo $ZONES
us-east-1aus-east-1bus-east-1cus-east-1dus-east-1eus-east-1
```

Without this line: 

```
ubuntu@ip-172-31-94-142:~$ echo $ZONES
us-east-1a us-east-1b us-east-1c us-east-1d us-east-1e us-east-1f
```

**Comment by arokem** [comment]
@consideRatio : see https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1237#discussion_r276057976. That additional line seems to be there to remove white-space between the zone names, but I think we need that white-space.

**Pull request merged**